cli/src/commands/content/updateChannelCensorshipStatus.ts

```diff
@@ -0,0 +1,77 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import chalk from 'chalk'
+import ExitCodes from '../../ExitCodes'
+
+export default class UpdateChannelCensorshipStatusCommand extends ContentDirectoryCommandBase {
+  static description = 'Update Channel censorship status (Active/Inactive).'
+  static flags = {
+    context: ContentDirectoryCommandBase.contextFlag,
+  }
+  static args = [
+    {
+      name: 'id',
+      required: false,
+      description: 'ID of the Channel',
+    },
+    {
+      name: 'status',
+      required: false,
+      description: 'New status of the channel (1 - active, 0 - inactive)',
+    },
+    {
+      name: 'rationale',
+      required: false,
+      description: 'rationale',
+    },
+  ]
+
+  async run() {
+    let { context } = this.parse(UpdateChannelCensorshipStatusCommand).flags
+
+    let { id, status, rationale } = this.parse(UpdateChannelCensorshipStatusCommand).args
+
+    if (!context) {
+      context = await this.promptForContext()
+    }
+
+    const currentAccount = await this.getRequiredSelectedAccount()
+    await this.requestAccountDecoding(currentAccount)
+
+    const actor = await this.getActor(context)
+
+    if (id === undefined) {
+      id = await this.promptForChannel()
+    }
+
+    if (status === undefined) {
+      status = await this.simplePrompt({
+        type: 'list',
+        message: 'Select new status',
+        choices: [
+          { name: 'Active', value: true },
```

Comment:
Perhaps `Censored` / `Not censored` or something like that would be more appropriate.
With this it can be a little confusing (whether the censorship is `Active` or the channel itself).

Reply:
Done here https://github.com/Joystream/joystream/pull/2280/commits/fa486b2d95db3817776525438ff8aeded0bc63d7